webui/react/src/services/apiConfig.ts

```diff
@@ -505,6 +505,20 @@ export const disableAgent: DetApi<string, Api.V1DisableAgentResponse, Type.Agent
   request: (agentId) => detApi.Cluster.disableAgent(agentId, {}),
 };
 
+export const getKubernetesResourceManagers: DetApi<
+  Service.GetKubernetesResourceManagersParams,
```

Comment:
i think you should be able to indicate that the request takes no parameters by using `void` as the type here.
```suggestion
  void,
```

Reply:
done